core/src/main/scala/org/typelevel/paiges/Doc.scala

```diff
@@ -619,7 +613,7 @@ object Doc {
    *
    * when we call `.grouped` on lineBreak we get lineOrEmpty
    */
-  val lineBreak: Doc = Line(false)
+  val lineBreak: Doc = FlatAlt(Line, Empty)
```

Comment:
you had an invariant that `FlatAlt(a, b)` has `isEmpty(a) == isEmpty(b)` above, but this is false here, right?

Reply:
Fixed.  Thanks for the catch.